pkg/client/dtclient/dummy_client.go

```diff
@@ -251,6 +251,10 @@ func (c *DummyClient) ListSchemas() (SchemaList, error) {
 	return make(SchemaList, 0), nil
 }
 
+func (d *DummyClient) Schema(schemaID string) (schema Schema, err error) {
+	return
```

Comment:
Please return empty values instead of using naked returns. I find it easier to see that something's returned instead of having to look up the return parameters.